generator/ServiceClientGeneratorLib/ServiceClientGeneratorLib.csproj

```diff
@@ -343,13 +343,8 @@
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
     </Compile>
-    <Compile Update="Generators/SourceFiles/ServiceClients.cs">
-      <DependentUpon>ServiceClients.tt</DependentUpon>
```

Comment:
`ServiceClients` and `ServiceInterface` were used for the 3.5 code. Removing them now doesn't impact the generated files anymore.